ratings/tests.py

```diff
@@ -54,10 +54,9 @@ def test_rating_change(self):
         """ Change your existing rating. """
         loggedin = self.client.login(username="testuser1", password="testpass")
         self.assertTrue(loggedin)
-        r = ratings.models.Rating.objects.create(object_id=self.sound.id, content_type=self.ct, user_id=self.user1.id, rating=4)
```

Comment:
This test is supposed to check that a rating for a user is updated if the rating already exists and then the user triggers the rating URL. If you delete this line then you are only checking that a rating is added, not updated